includes/class-wc-calypso-bridge-tracks.php

```diff
@@ -45,6 +54,26 @@ private function __construct() {
 		add_filter( 'pre_option_woocommerce_allow_tracking', array( $this, 'always_enable_tracking' ) );
 	}
 
+	/**
+	 * Set's the value for the tracks host property.
+	 */
+	public function set_tracks_host_value() {
+		// Default value assumes business plan, inside wp-admin.
+		$host_value = 'bizplan-wp-admin';
+
+		// If an ecomm plan site, update host value.
+		if ( wc_calypso_bridge_is_ecommerce_plan() ) {
+			if ( 1 === (int) get_user_meta( get_current_user_id(), 'calypsoify', true ) ) {
```

Comment:
When I run this branch, `get_current_user_id()` here returns `0`, when the current user is `1`, which causes the wrong user meta to be looked up. I think this is because this code is executing before the current user is set.

I think the solution is to wait to call `get_instance` until the `init` action fires. So, instead of...

```
$wc_calypso_bridge_setup = WC_Calypso_Bridge_Tracks::get_instance();
```

do...

```
add_action( 'init', array( 'WC_Calypso_Bridge_Tracks', 'get_instance' ) );
```

I tested this, and it seems to do the trick.

Reply:
Thanks @mattsherman - sorry this wasn't working... and wondering why it was working for me locally. I applied a change per your suggestion and it is testing out well for me, thanks!